pkg/resource/plugin/analyzer_plugin.go

```diff
@@ -606,16 +619,23 @@ func constructEnv(opts *PolicyAnalyzerOptions) ([]string, error) {
 		}
 	}
 
+	langSpecificVar := func(k string) string {
+		if runtime == "nodejs" {
+			return strings.Replace(k, "PULUMI_", "PULUMI_NODEJS_", 1)
+		}
+		return k
+	}
```

Comment:
Possibly not now - but I wonder if we want to transition to using consistent names, and pass both to Node.js for now, and start reading from both in the Node.js implementation so that we can get to a consistent pattern here soon and not need Node specific code at this layer?